wm/desk.go

```diff
@@ -209,6 +209,13 @@ func (x *x11WM) runLoop() {
 				if c.(*client).id == ev.Event {
 					if x.moveResizing {
 						x.moveResizeEnd()
+						// ensure menus etc update
```

Comment:
This is doubling up the previous fix - should remove the other fix.
And would it not be neater to put this inside moveResizeEnd rather than in this runloop, which is already quite long.